FIWARE-adapter/naiades-data-docs-app/src/environments/environment.prod.ts

```diff
@@ -0,0 +1,5 @@
+export const environment = {
+  apiUrl: 'localhost:5004',
```

Comment:
Can we add this stuff into some .env file?

Reply:
It is already in an angular default environment file, which I think is better than `.env` so we don't need to use `dotenv`...